src/DigitalSubAuth.js

```diff
@@ -0,0 +1,98 @@
+const AWS = require('aws-sdk')
+const moment = require('moment')
+
+AWS.config.update({
```

Comment:
Double check if you really need to do this, I was able to remove it on my Lambda